README.md

```diff
@@ -1,6 +1,6 @@
 # Mini LSM Tree Database System
 
-This C project represents a simplified version of a log structed merge (LSM) tree database system. 
+This C project represents a simplified version of a log-structed merge (LSM) tree database system. 
```

Comment:
I'd just say `This C project is a simplified...`

Reply:
👍